modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/podcast/EpisodeListAdapter.kt

```diff
@@ -76,7 +76,11 @@ class EpisodeListAdapter(
                 playbackStateUpdates = playbackManager.playbackStateRelay,
                 upNextChangesObservable = upNextQueue.changesObservable,
                 imageLoader = imageLoader,
-                swipeButtonLayoutFactory = swipeButtonLayoutFactory
+                swipeButtonLayoutFactory = swipeButtonLayoutFactory,
+                userBookmarksObservable = bookmarkManager.findPodcastBookmarksFlow(
+                    podcastUuid = fromListUuid ?: "",
```

Comment:
⚠️ `fromListUuid` is not a `podcastUuid`. We need to get it from the podcast episode.

Reply:
okay